modules/integration/src/test/scala/scala/cli/integration/ScalafixTests.scala

```diff
@@ -0,0 +1,59 @@
+package scala.cli.integration
+
+import com.eed3si9n.expecty.Expecty.expect
+
+class ScalafixTests extends ScalaCliSuite {
+  override def group: ScalaCliSuite.TestGroup = ScalaCliSuite.TestGroup.First
+
+  val confFileName = ".scalafix.conf"
+
+  val emptyInputs: TestInputs = TestInputs(os.rel / ".placeholder" -> "")
+
+  val simpleInputsUnformattedContent: String =
```

Comment:
Referring to unformatted /formatted content is a bit confusing as scalafix is not a formatting tool. I would suggest original/rewritten.